// for testnet

export const VITE_COINFLIP_ADDRESS ="0xe5209b43Db1508f7943b505889A07122Ae5e83da"

// "0x54B35AaF91B6B50C6ECf3187c4031C4c9518cB4B" working fine except weird bug
//'0x7745D3ACf8F66460687606d8500BAD660fc7Ef48" working on prod but not ideal.


export const VITE_RPC_URL="https://zksync2-testnet.zksync.dev"
export const VITE_BLOCK_EXPLORER_URL="https://zksync2-testnet.zkscan.io/"
export const VITE_NETWORK_ID = 280 // 324 for mainnet
export const VITE_NETWORK_STRING = "zkSync Era Testnet"



export const ALCHEMY_ID = "SGZRXaj2JYspAeKmgkhiPpyRHeC_muhA"

// // for mainnet

// export const VITE_COINFLIP_ADDRESS = ""


// export const VITE_RPC_URL= import.meta.env.VITE_RPC_URL || "https://zksync2-mainnet.zksync.io"
// export const VITE_BLOCK_EXPLORER_URL= import.meta.env.VITE_BLOCK_EXPLORER_URL || "https://explorer.zksync.io/"
// export const VITE_NETWORK_ID = import.meta.env.VITE_NETWORK_ID || 324
// export const VITE_NETWORK_STRING = import.meta.env.VITE_NETWORK_STRING || "zkSync Era Mainnet"
